lib/APPTurtle/src/App.cpp

```diff
@@ -122,9 +132,52 @@ void App::setup()
         {
             LOG_ERROR("Network configuration could not be set.");
         }
+        else if (false == m_ros.setConfiguration(settings.getRobotName(), "", settings.getMqttBrokerAddress(),
+                                                 settings.getMqttPort()))
+        {
+            LOG_ERROR("Micro-ROS Agent could not be configured.");
+        }
         else
         {
-            isSuccessful = true;
+            /* Create the Subscriber Callback. */
+            TwistSubscriber::RosTopicCallback twistCallback = [this](const geometry_msgs__msg__Twist* msgData)
+            {
+                if (nullptr == msgData)
+                {
+                    LOG_ERROR("TwistCallback received nullptr.");
+                }
+                else
+                {
+                    LOG_DEBUG("Twist data received.");
+
+                    /* Short blink to indicate reception. */
+                    Board::getInstance().getBlueLed().enable(true);
+                    delay(50U);
```

Comment:
Should we really delay ? 
We could turn it off when we finishedf processing the message.

Reply:
Fixed